src/main/scala/kamon/logback/instrumentation/AsyncAppenderInstrumentation.scala

```diff
@@ -29,10 +29,13 @@ import scala.beans.BeanProperty
 
 object AsyncAppenderInstrumentation {
 
-  val MdcTraceKey : String = "kamonTraceID"
-  val MdcSpanKey : String = "kamonSpanID"
+  @volatile private var _mdcContextPropagation: Boolean = true
+  @volatile private var _mdcTraceKey: String = "kamonTraceID"
+  @volatile private var _mdcSpanKey: String = "kamonSpanID"
 
-  @volatile var mdcContextPropagation : Boolean = true
```

Comment:
as far as i can see and @ivantopo can confirm, private variables in other kamon project don't use "_" so for consistency purposes I'd recommend to align with code convention and remove _ 
this is more for consistency rather than personal preferences

Reply:
Hi, @milenkovicm - first (and main) class I can find: https://github.com/kamon-io/Kamon/blob/master/kamon-core/src/main/scala/kamon/Kamon.scala and "_" ;)

Reply:
apparently they are not really consistent https://github.com/kamon-io/kamon-prometheus/blob/master/src/main/scala/kamon/prometheus/PrometheusReporter.scala 
https://github.com/kamon-io/kamon-akka/blob/master/kamon-akka-2.5.x/src/main/scala/kamon/akka/Akka.scala

:) 

let the boss @ivantopo decide what he prefers :)

Reply:
I totally see the consistency issue here. We can agree to be consistently inconsistent! I think it's time for us to get a bit more serious on coding conventions but that's far beyond the scope of this PR.

Reply:
Btw, now being serious on this. The `mdcContextPropagation` variable should be private and only exposed through a getter. That's the case in which we mostly end up with a private `_` preffix: we want this to be a var, we want it to be accessible through a nice name like `mdcContextPropagation`, but we don't want to allow anyone to write to this value. It should only be modified from a `reconfigure` hook and nowhere else.

Please make it private :)

Reply:
@ivantopo _mdcContextPropagation is private, mdcContextPropagation is its getter. Unless you took a look at previous version I can't understand what is to be changed now ;)

Reply:
Oh man, I'm sorry. I was looking at the removed lines instead of the new ones :sob:... going to refill the coffee!

Reply:
The other change on the configuration setting names above, that one would be nice to have!

Reply:
@ivantopo you mean https://github.com/kamon-io/kamon-logback/pull/7#discussion_r175778963 ?

Reply:
correct!